### PR TITLE
Feature Request: Bunny crate

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

A common player on manual recently requested the ability to more easily have rabbits as crewmembers. This feature introduces a reasonably priced rabbit crate to cargo to amend it.

## Why It's Good For The Game

Bunnies are cool I guess.  

## Changelog

:cl:
add: Introduces orderable rabbit crate to cargo livestock.
/:cl:

### CODE REVIEW

**Comment by san7890** [comment]
I've been wanting to do this since #64175, thanks for doing it for me.

File: code/modules/cargo/packs/livestock.dm [hunk 1]
```diff
@@ -146,6 +146,13 @@
 	contains = list(/mob/living/simple_animal/hostile/retaliate/goat)
 	crate_name = "goat crate"
 
+/datum/supply_pack/critter/rabbit
+	name = "Rabbit Crate"
+	desc = "What noise do rabbits even make? Contains one rabbit."
```
Comment (san7890) [inline code/modules/cargo/packs/livestock.dm:151]: https://github.com/tgstation/tgstation/blob/wall-listener/code/datums/ai/basic_mobs/basic_subtrees/speech_subtree.dm#L63

**Comment by Iamgoofball** [comment]
bnuuy

**Comment by Mothblocks** [comment]
That person on manuel should've learned how to do this themselves, you don't need to spoonfeed them

File: code/modules/cargo/packs/livestock.dm [hunk 2]
```diff
@@ -146,6 +146,13 @@
...
+	cost = CARGO_CRATE_VALUE * 4
+	contains = list(/mob/living/basic/rabbit)
+	crate_name = "rabbit crate" //I wanted to call it the bunny box
```
Comment (Mothblocks) [inline code/modules/cargo/packs/livestock.dm:154]:
```suggestion
	crate_name = "rabbit crate"
```

This comment isn't important

Comment (Iamgoofball) [reply code/modules/cargo/packs/livestock.dm]: and yours is?

**Comment by DemoCrack** [comment]
bnuuy

**Comment by AviiCinders** [comment]
I can finally stop using the pray command for bnnuy

**Review by ArcaneMusic — APPROVED** [review]
Looks clean, cost's fine, glad to see it

**Pull request merged**